### PR TITLE
[KARAF-5712] Change responsive css to Bootstrap

### PR DESCRIPTION
**Pull request by fpapon**
Upgrade to Bootstrap CSS 4
Add examples section
Add Karaf Slack link

### CODE REVIEW

**Comment by jbonofre** [comment]
I guess this PR include #8 and #7 right ?

**Comment by fpapon** [comment]
Yes :)

File: src/main/webapp/community.html [hunk 1]
```diff
@@ -1,115 +1,161 @@
...
+              </h4>
+              <p class="card-text"> [ <a href="https://builds.apache.org/view/H-L/view/Karaf/" target="_blank">visit</a> ]</p>
+
+              <h4 class="mb-0">
+                <a class="text-dark">#apache-karaf on Freenode</a>
+              </h4>
+              <p class="card-text"> [ <a href="irc://irc.freenode.net/apache-karaf" target="_blank">visit</a> ]</p>
+
+              <h4 class="mb-0">
+                <a class="text-dark">#karaf sur the-asf.slack.com</a>
```
Comment (jbonofre) [inline src/main/webapp/community.html]: Please use english wording: `on` instead of `sur`.

File: src/main/webapp/community.html [hunk 2]
```diff
@@ -120,82 +166,97 @@ <h3 id="pull-request">Provide changes in a pull request</h3>
...
+        </div>
 
-                <h3 id="review-pull-request">Review pull requests</h3>
-                
-                Anyone can review pull requests but the merge can be done only by a committer.<br/>
+        <div class="row mb-5 mt-5">
+            <div class="col">
+              <h4 class="pb-3 mb-4">Review pull requests</h4>
+              <p>Anyone can review pull requests but the merge can be done only by a committer.<br/>
                 If you want to review a PR then comment with <code>R:@yourgithubid</code>. This marks that you will do the review and also apply if you are a committer.<br/>
```
Comment (jbonofre) [inline src/main/webapp/community.html]: We can mention the "Reviewer" entry on the right menu, allowing a committer to be reviewer (no need to use `R:` in that case).

File: src/main/webapp/community.html [hunk 3]
```diff
@@ -120,82 +166,97 @@ <h3 id="pull-request">Provide changes in a pull request</h3>
...
+                        <code>> git rebase -i apache/master</code>
+                    </li>
                     <li>Merge the PR (it automatically closes the PR) :<br/>
-                        <code>git fetch --all</code><br>
-                        <code>git checkout apache/master</code><br>
-                        <code>git merge --no-ff -m "[KARAF-nnnn] This closes #xxx" pr-xxx</code><br>
-                        <code>git push apache HEAD:master</code><br>
+                        <code>> git fetch --all</code><br>
+                        <code>> git checkout apache/master</code><br>
+                        <code>> git merge --no-ff -m "[KARAF-nnnn] This closes #xxx" pr-xxx</code><br>
```
Comment (jbonofre) [inline src/main/webapp/community.html]: That's not the preferred way: we now can use the merge button on github (for committer).

**Review by jbonofre — CHANGES_REQUESTED** [review]

**Comment by fpapon** [comment]
Changes done, can you review ?

**Review by jbonofre — APPROVED** [review]

**Pull request merged**